### ISSUE TITLE
Is kubernetes version 1.23 supported?

### ISSUE THREAD

**Issue by EricNexus** [minio/directpv#507]
Running kubectl directpv install
get:
I0113 11:54:54.820942  265762 install.go:93] Enable dynamic drive change management using --enable-dynamic-discovery flag
I0113 11:54:54.821075  265762 install.go:94] This flag will be made default in the next major release version
ERROR unsupported kubernetes version 1.23

**Comment by luckydogxf** [comment]
Seems `Not`.
I ran into the same issue, and I downgrade my `kubeadm` and `kubectl` to 1.22. It works.

**Comment by Praveenrajmani** [comment]
yes, it is not supported yet. will add the support in the next release.

**Comment by k963** [comment]
How about support for EKS cluster, What's the required version? , I tried with both v 1.18 and v1.21. It says unsupported version for both of these.
![image](https://user-images.githubusercontent.com/14110807/153136109-89b7bbd3-e0c6-4c52-b28a-f8549846c54d.png)

**Comment by harshavardhana** [comment]
This looks like a parsing error. the version string has an odd `18+`

**Comment by k963** [comment]
But it's from AWS EKS, How can I suppress that, in that case it has for v1.21 also.

**Comment by harshavardhana** [comment]
> I have used 1.18 and 1.21 in the yaml file respectively, even AWS console shows only 1.18.

Yeah because they broke the Minor version format @k963 here by introducing their own version string. We need to relax this check on our end.  For now, I don't think directpv will work on EKS.

**Comment by k963** [comment]
@harshavardhana Thanks for the update, will look for future release.

**Comment by wlan0** [comment]
@k963 We have pushed a fix for this issue. We will update here once we make a release.

**Comment by wlan0** [comment]
@k963 @EricNexus @luckydogxf https://github.com/minio/directpv/pull/534 PR fixes installation on eks. 

NOTE: The issue with invalid semver format in eks is documented here: https://github.com/aws/containers-roadmap/issues/1404

**Comment by vanlueckn** [comment]
Are there any news about the release with support for 1.23?

**Comment by amirkool** [comment]
> Are there any news about the release with support for 1.23?

any news on this (support for 1.23)?
